flow/actorcompiler/ActorCompiler.cs

```diff
@@ -353,8 +356,6 @@ public void Write(TextWriter writer)
             }
             bodyContext.catchFErr.WriteLine("loopDepth = 0;");
 
-            if (isTopLevel) writer.WriteLine("namespace {");
```

Comment:
Turns out you can't friend a class in an anonymous namespace. I think this is ok though since if we violate the ODR with these symbols then we also would have violated ODR with the actor itself